doc/source/tune/api/stoppers.rst

```diff
@@ -0,0 +1,41 @@
+.. _tune-stoppers:
+
+Tune Stopping Mechanisms (tune.stopper)
+=======================================
+
+In addition to Trial Schedulers like :ref:`ASHA <tune-scheduler-hyperband>`, where a number of
+trials are stopped if they perform subpar, Ray Tune also supports custom stopping mechanisms to stop trials early. They can also stop the entire experiment after a condition is met.
+For instance, stopping mechanisms can specify to stop trials when they reached a plateau and the metric
+doesn't change anymore.
+
+Ray Tune comes with several stopping mechanisms out of the box. For custom stopping behavior, you can
+inherit from the :class:`Stopper <ray.tune.Stopper>` class.
+
+Other stopping behaviors are described :ref:`in the user guide <tune-stopping-ref>`.
+
+
+.. _tune-stop-ref:
+
+Stopper Interface (tune.Stopper)
+--------------------------------
+
+.. currentmodule:: ray.tune.stopper
+
+.. autosummary::
+    :toctree: doc/
+
+    Stopper
```

Comment:
Can we explicitly set this include `_call_` as was done previously?

[Currently](https://ray--32311.org.readthedocs.build/en/32311/tune/api/doc/ray.tune.stopper.Stopper.html#ray.tune.stopper.Stopper) this shows `__init__` instead and feels a bit mismatched.

Reply:
I don't think it's actually possible to include specific members with the `autosummary`. Best thing I could do was have it right below.